assets/src/js/enhancedDonateButton.js

```diff
@@ -1,34 +1,58 @@
-export const setupEnhancedDonateButton = function() {
-  function isMobile() {
-    return window.matchMedia('(max-width: 576px)').matches;
-  }
+export const setupEnhancedDonateButton = () => {
+  const isMobile = () => window.matchMedia('(max-width: 576px)').matches;
 
-  function setupDonateButton() {
+  const setupDonateButton = () => {
     if ( isMobile() ) {
-      if (!$('.btn-enhanced-donate.btn-donate-top').length) {
-        $('.btn-enhanced-donate').clone().appendTo('body').addClass('btn-donate-top');
-        $('.btn-donate-top').parent('body').addClass('with-donate-on-top');
+      if ( ! document.querySelector('.btn-enhanced-donate.btn-donate-top') ) {
+        const enhancedDonateBtn = document.querySelector('.btn-enhanced-donate').cloneNode(true);
```

Comment:
Shouldn't we first check that `document.querySelector('.btn-enhanced-donate')` exists? 🤔

Reply:
yes, here we need to check `document.querySelector('.btn-donate-top')`, PR updated :)

Reply:
Oh no that's not what I meant, sorry 🙂 I meant we should check that `document.querySelector('.btn-enhanced-donate')` exists before cloning it, but now I see that you already check that in `ready` function so it's fine ✌️